v19.1/rename-database.md

```diff
@@ -25,6 +28,27 @@ Parameter | Description
 ----------|------------
 `name` | The first instance of `name` is the current name of the database. The second instance is the new name for the database. The new name [must be unique](#rename-fails-new-name-already-in-use) and follow these [identifier rules](keywords-and-identifiers.html#identifiers). You cannot rename a database if it is set as the [current database](sql-name-resolution.html#current-database) or if [`sql_safe_updates = true`](set-vars.html).
 
+## Database renaming considerations
+
+Database renames are not transactional.
+
+There are two phases during a database rename:
+
+1. The database descriptor (an internal data structure) is updated, and announced to every other node.
+2. The `system.namespace` table is updated.
+
+The name of a database is both in the descriptor and the `system.namespace` table.
+
+Step 1 is non-transactional. The rename will be announced to other nodes even if the transaction is aborted.
```

Comment:
the rename will be announced only if the transaction commits but there is no guarantee on the time this is going to take.

Reply:
Fixed by updating with that info.  Also rewrote this section a bit to move the info about these steps all into one place.  Now it looks like:

> There are two phases during a rename:
> 1. The `system.namespace` table is updated.  This phase is transactional, and will be rolled back if the transaction aborts.
> 2. The table descriptor (an internal data structure) is updated, and announced to every other node. This phase is **not** transactional. The rename will be announced to other nodes only if the transaction commits, but there is no guarantee on how much time this operation will take.